build.assets/buildbox/Dockerfile

```
@@ -85,11 +85,10 @@ FROM base AS rust
 RUN install -d -m 0775 -o buildbox -g buildbox /opt/rust
 USER buildbox
 
-ARG RUST_VERSION
 ENV RUSTUP_HOME=/opt/rust
 ENV CARGO_HOME=/opt/rust
 RUN curl --proto =https --tlsv1.2 -fsSL https://sh.rustup.rs | \
-	sh -s -- -y --profile minimal --default-toolchain ${RUST_VERSION} && \
+	sh -s -- -y --profile minimal && \
```

I think this can be one step and then we can COPY the `rust-toolchain.toml` file somewhere to then `RUN rustup install` to install the correct toolchain with components and targets from the config file, with the understanding that if it changes between the image being built and the execution, we'll only suffer some additional downloads.

```suggestion
	sh -s -- -y --profile minimal --default-toolchain none
```

---

You lost me on the COPY step.